addon/webextension/experiments/screenshots/api.js

```diff
@@ -0,0 +1,42 @@
+/* globals AppConstants */
+
+"use strict";
+
+ChromeUtils.defineModuleGetter(this, "AppConstants",
+                               "resource://gre/modules/AppConstants.jsm");
+ChromeUtils.defineModuleGetter(this, "Services",
+                               "resource://gre/modules/Services.jsm");
+
+this.screenshots = class extends ExtensionAPI {
+  getAPI() {
+    return {
+      experiments: {
+        screenshots: {
+          // If you are checking for 'nightly', also check for 'nightly-try'.
+          //
+          // Otherwise, just use the standard builds, but be aware of the many
+          // non-standard options that also exist (as of August 2018).
+          //
+          // Standard builds:
+          //   'esr' - ESR channel
+          //   'release' - release channel
+          //   'beta' - beta channel
+          //   'nightly' - nightly channel
+          // Non-standard / deprecated builds:
+          //   'aurora' - deprecated aurora channel (still observed in dxr)
+          //   'default' - local builds from source
+          //   'nightly-try' - nightly Try builds (QA may occasionally need to test with these)
+          async getUpdateChannel() {
```

Comment:
Do these all need to be async?

Reply:
Nope, I copied [the examples](https://webextensions-experiments.readthedocs.io/en/latest/basics.html#how-do-they-work) a little too closely. Since experimental APIs extend the same base `ExtensionAPI` class as built-in APIs, and the `async` bit is definitely [not required there](https://searchfox.org/mozilla-central/source/toolkit/components/extensions/parent/ext-telemetry.js#30), must not be required here, either.